README.md

```diff
@@ -1,634 +1,1855 @@
+assets are maintained, and contract upgrades occur democratically. This protocol natively supports
+*all* Cardano assets as both loan assets and collateral assets - even assets that have yet to be
+created.
 
-##### Distinguishing features specific to Cardano-Loans:
-
-1. **On-Chain Credit History** - the status/conditions of current & past loans associated with a borrower's stake pubkey are easily queryable by prospective lenders and third-party data miners.
-2. **Trustless Negotiations** - all loan conditions are negotiated in a fully p2p fashion - interest rates, collateral (token type(s) and relative prices), and length of the loan - all are negotiable parameters. Multiple tokens can be used as collateral, even NFTs. Over/under collateralization is implied by the relative prices in the lender's offer, which is impacted by the borrower's credit history.
-3. **Partial Repayments** - borrowers can repay loans incrementally and withdraw collateral in proportion to the repayment. If an outstanding balance remains at expiry, the lender may withdraw the remaining collateral. 
-4. **No Auto-liquidation** - All loan terms (including relative token prices) are agreed upon explicitly by both parties, so "margin" is constant throughout the length of the loan, and does *not* change with global price movements. 
-5. **Endogenous Price & Interest Rate Discovery** - instead of relying on oracles or trusted actors, relative token prices and interest rates are explicitly agreed upon during loan negotiations, resulting in true market-driven discovery. In due time and with enough users, sufficiently distributed dApps may *serve* as oracles, instead of having to consume them.
-
-
-##### Features that Cardano-Loans shares with other *distributed dApp* protocols:
-
-- **Full Custody** - users always maintain full spending *and* delegation control over their assets.
-- **Natural Concurrency** - throughput scales *with* the number of users. No specialized batchers/indexers are necessary, though they *may* be used in proprietary backends of large lending providers.
-- **Frontend/Backend Agnosticism & Interoperability** - relatively straightforward integration with existing frontends (i.e. wallets). Third-party platforms can build business models atop open-contract p2p protocols. 
-- **Censorship Resistance** - due to the aforementioned agnosticism, distributed p2p dApps offer the highest level of censorship-resistance of any dApp architecture.
-- **No Superfluous "dApp" Tokens** - ADA is all you need to pay for script/TX fees. 
-- **Democratic Upgradability** - users choose if/when to use new contracts.
+## Motivation
 
+A healthy credit-debt market is a vital (and often the largest) component of a thriving economy. A
+healthy market is one in which prices reflect underlying reality as quickly and faithfully (with as
+little friction) as possible. For the sake of avoiding a whole treatise on economics, suffice to say
```

Comment:
This is really a personal style issue and therefore you may not agree, but I think it might read better with commas instead of (brackets) in a few areas.  IE: Users deploy and interact with each others' script addresses.  ... as quickly, and faithfully, with as little friction as possible.

Reply:
I agree with the first two but the last one is meant to be a definition of "faithfully". Therefore, for that one, I'll change that one to: "(ie, with as little friction)".